src/models/simple_metabolism_model.py

```diff
@@ -135,22 +130,21 @@ def get_iob_from_sbr(self, sbr_actual):
         num_hours_post_t0 = 8
         minutes_per_pump_pulse = 5
```

Comment:
I would consider renaming this `minutes_per_pump_pulse` to something like `simulation_interval,` but you are correct that this should default to 5 minutes given that is what Loop does.

Reply:
My thinking with this was if we wanted to eventually have this simulate different pumps in this function or eventual related functions. In that case this variable would be a property of a pump class. But if we don't want to get that abstract yet, I'm fine changing the name.